app/customizer.html

```diff
@@ -164,7 +164,7 @@
             </div>
           </div>
 
-          <div id="customizer-viewer"></div>
```

Comment:
Using same Id as in app.html to allow ThreeViewer work with flood and customizer.